### DISCUSSION TITLE
Any recommendations for showing captions for audio using media-chrome library?

### DISCUSSION THREAD

**Discussion by hellovietduc** [muxinc/media-chrome#937] [Q&A] [answered]
I'm using media-chrome to build a custom **audio player**. It's very configurable in terms of styling so it's a great choice for me. However, I have yet to find a guide or suggestion for how to display captions with this library. I'm aware that `<track>` element when put inside `<audio>` won't display captions so I have tried changing to `<video>` but I think media-chrome hides the native `<video>` element inside shadow DOM so the captions aren't visible. 

Did anyone try using this library to build an audio player that shows captions? Any recommendations for me? Thanks.

**Comment by tomashforth** [comment]
Not so much an answer as an addition: facing similar issues trying to add 
track elements to youtube-video...

**Comment by hellovietduc** [comment, marked as answer]
I ended up using this https://github.com/vidstack/captions

Works well for me so far.
